src/content/news/2023-06-30-lectureship.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 2023 “陳道明牧師”神學講堂
+date: 2023-09-16
```

Comment:
The date is not shown in the website. It is only used to record the post date and the news are sorted by date. You probably want to change it to the post date to be consistent with the date in your file name. If you set the date too far ahead, new news items will be sorted behind this one. That may not be the expected behavior.